app/services/employee_importer.rb

```diff
@@ -0,0 +1,74 @@
+require "slack-ruby-client"
+
+class EmployeeImporter
+  def initialize
+    configure_slack
+  end
+
+  def import(dry_run = false)
+    import_results = {created: 0, skipped: 0, dry_run: false}
```

Comment:
is our rubocop config enforcing no spaces between curly braces? I thought I'd copied over the tbot guide but maybe something went wrong...